wdtk-wikibaseapi/src/main/java/org/wikidata/wdtk/wikibaseapi/apierrors/MediaWikiApiErrorHandler.java

```diff
@@ -33,6 +33,7 @@
 	public final static String ERROR_INVALID_TOKEN = "badtoken";
 	public final static String ERROR_NO_SUCH_ENTITY = "no-such-entity";
 	public final static String ERROR_MAXLAG = "maxlag";
+	private static final String ERROR_ASSERT_USER_FAILED = "assertuserfailed";
```

Comment:
It should be package private to be able to reuse it in AssertUserFailedException definition.

Reply:
Oops, sorry! I have made it public to match the rest of the codes above.